src/components/Button.jsx

```diff
@@ -52,10 +52,14 @@ const StyledButton = styled.button`
 `;
 
 export const Button = (props) => (
```

Comment:
It will be cleaner if you will use destructing here

Reply:
fixed in the next task